projects/go-lib/src/lib/components/go-copy/go-copy.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  animations: [],
+  selector: 'go-copy',
+  templateUrl: './go-copy.component.html',
+  styleUrls: ['./go-copy.component.scss']
+})
+export class GoCopyComponent {
+  @Input() text: string;
+
+  constructor() { }
+
+  copyStringToClipboard (element: HTMLInputElement) {
```

Comment:
There isn't a return type on this function. If we want to make the most of typescript we should add one:
```suggestion
  copyStringToClipboard(element: HTMLInputElement): void {
```